tests/karma/search/containers/TestAddonPage.js

```diff
@@ -19,20 +47,150 @@ describe('AddonPage', () => {
     ));
   }
 
-  it('renders the name', () => {
+  describe('rendered fields', () => {
     const root = render({state: initialState, props: {params: {slug: 'my-addon'}}});
-    assert.equal(root.querySelector('h1').textContent, 'Addon!');
+
+    it('renders the name', () => {
+      assert.equal(root.querySelector('h1').textContent, 'Addon!');
+    });
+
+    it('renders the summary', () => {
+      assert.equal(root.querySelector('.addon--summary').textContent, 'My add-on');
+    });
+
+    it('renders the description', () => {
+      assert.equal(
+        root.querySelector('.addon--description').textContent, 'An add-on that adds on.');
+    });
+
+    it('renders the tags', () => {
+      const tags = root.querySelector('.addon--tags').childNodes;
```

Comment:
couldn't this be `querySelectorAll('.addon--tags div')` or something? I can't actually find the `addons--tags` class anywhere so I don't know what the child nodes are.